examples/controls_example/common/options_list/is_valid_search.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { getIpRangeQuery, getIsValidFullIp } from './ip_search';
+import { OptionsListSearchTechnique } from './suggestions_searching';
+
+/**
```

Comment:
This comment is no longer needed since now entire factory is async imported in the new framework